configuration/configmgr/configmgrlib/EnvironmentMgr.cpp

```diff
@@ -68,18 +72,21 @@ bool EnvironmentMgr::loadSchema(const std::string &configPath, const std::string
         }
 
         //
-        // Load any support libs that are environment specific
-        auto findIt = cfgParms.find("support_libs");
-        if (findIt != cfgParms.end())
+        // Load support libs based on the schema type which is read from the itemType property of the schema root node.
+        std::string envType = m_pSchema->getItemType();
+        std::string libPath = LIB_DIR;
+        std::string libMask = "libcfg" + envType + "_*";
```

Comment:
A subdir could be used to identify the libs to load, and might be more reliable. Would there be an issue with using a subdir instead?